src/Benchmarks/Configuration/Scenarios.cs

```diff
@@ -41,12 +41,18 @@ public Scenarios(IScenariosConfiguration scenariosConfiguration)
         [ScenarioPath("/json")]
         public bool Json { get; set; }
 
+        [ScenarioPath("/jil")]
```

Comment:
New scenarios must also be added to `src/Benchmarks.ServerJob/Scenario.cs` and `src/BenchmarksDriver/Program.cs`.  See https://github.com/aspnet/benchmarks/pull/301/files.

Reply:
Done.